Product/Production/Common/CONNECTCommonWeb/src/main/java/gov/hhs/fha/nhinc/connectmgr/uddi/UDDIUpdateManagerHelper.java

```diff
@@ -190,6 +197,25 @@ public UDDIUpdateManagerForceRefreshResponseType forceRefreshFileFromUDDIServer(
 
         return oResponse;
     }
+    
+    private void addLastUpdateTime(BusinessDetail businessDetail) {
+        try {
+            if(PropertyAccessor.getInstance().getPropertyBoolean(GATEWAY_PROPERTY_FILE, UDDI_UPDATE_TIME)) {
+                businessDetail.setUddiUpdatetime(getTimestamp());
+            }
+        } catch (PropertyAccessException ex) {
+            LOG.error("Unable to access property: " + UDDI_UPDATE_TIME, ex);
```

Comment:
Optional: Recommend to use parameterized logging such as LOG.error("Unable to access property: {}, {}"",UDDI_UPDATE_TIME, ex);